### DISCUSSION TITLE
Stop print on lcd when host printing

### DISCUSSION THREAD

**Discussion by toothyc** [Jyers/Marlin#81] [Q&A] [answered]
Hi, Is the stop print button the LCD meant to stop the print from octoprint? Mine doesn't and continues to print after clicking stop.

**Comment by t3chguy** [comment, marked as answer]
You must enable `HOST_ACTION_COMMANDS` in Marlin's configuration.

**Reply by toothyc** [reply]
thank for the quick response, i will try flashing and report back

**Reply by toothyc** [reply]
@t3chguy that worked a treat thank you
